openstack_exporter/__init__.py

```diff
@@ -0,0 +1,4 @@
+from pbr import version
+
+version_info = version.VersionInfo('openstack_exporter')
```

Comment:
Where is the actual Version info maintained?

Reply:
The versioning for PBR is in git tags.
https://docs.openstack.org/pbr/latest/user/features.html#version